Updated Instructions

iogburu opened the pull request:
<!-- FreeCodeCamp Pull Request Template -->

<!-- IMPORTANT Please review https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md for detailed contributing guidelines -->

<!-- Help with PRs can be found at https://gitter.im/FreeCodeCamp/HelpContributors -->

<!-- Make sure that your PR is not a duplicate -->
#### Pre-Submission Checklist

<!-- Go over all points below, and put an `x` in all the boxes that apply. -->

<!-- All points should be checked, otherwise, read the CONTRIBUTING guidelines from above-->

<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] Your pull request targets the `staging` branch of FreeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [X] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

<!-- What type of change does your code introduce? Put an `x` in the box that applies. -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

<!-- Go over all points below, and put an `x` in all the boxes that apply. -->

<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] Tested changes locally.
- [ ] Closes currently open issue (`Closes #XXXX`): Closes 
#### Description

<!-- Describe your changes in detail -->

Updated Instructions for "Use Responsive Design with Bootstrap Fluid Containers" section in order to inform users who may jump around in the tutorials where they can learn how to build the Cat Photo App.

iogburu committed:
Updated Instructions

Updated Instructions for "Use Responsive Design with Bootstrap Fluid Containers" section in order to inform users who may jump around in the tutorials where they can learn how to build the Cat Photo App.

theflametrooper commented on seed/challenges/01-front-end-development-certification/bootstrap.json:
```diff
@@ -8,7 +8,7 @@
       "title": "Use Responsive Design with Bootstrap Fluid Containers",
       "description": [
-        "Now let's go back to our Cat Photo App. This time, we'll style it using the popular Bootstrap responsive CSS framework.",
+        "In the HTML5 and CSS section of FreeCodeCamp we built a Cat Photo App. Now let's go back to our Cat Photo App. This time, we'll style it using the popular Bootstrap responsive CSS framework.",
```
Repetitive use of `Cat Photo App`, "we built a Cat Photo App. Now let's go back to our Cat Photo App."

iogburu replied:
This update was made, as not everyone will have performed the previous activity. The line was added to ensure participants understand where the previous code came from.

theflametrooper replied:
Try and reword the sentences so you don't use the word `CatPhotoApp` more than once.

iogburu committed:
fix/ Updated Instructions/Reduced Redundancy

This update was made, as not everyone will have performed the previous activity. The line was added to ensure participants understand where the previous code came from.

iogburu commented:
@theflametrooper Cat Photo App has been redundancy has been removed.

theflametrooper commented:
@iogburu Please squash your commits

raisedadead commented:
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @iogburu and @theflametrooper .
